src/js/toolbar.js

```diff
@@ -0,0 +1,16 @@
+//@deno-types=./19.ts
+import { tag } from "./19.js"
+import { validate } from "./validate.js"
+import { FocusGroupMixin } from "./focusgroup.js"
+
+// TODO: Support focusability of disabled elements
```

Comment:
My understanding is that the proper way to allow focusability of disabled elements is to use `[aria-disabled]` instead of `[disabled]`. This might involve some changes to `FocusGroupMixin`.